data/maps/SevenIsland_PokemonCenter_1F/text.inc

```diff
@@ -6,7 +6,7 @@ SevenIsland_PokemonCenter_1F_Text_EventsDependingOnConnections::
     .string "your own.$"
 
 SevenIsland_PokemonCenter_1F_Text_DidYouRideSeagallopHere::
-    .string "Did you ride a SEAGALLOP ferry\n"
+    .string "Did you ride a ALOLAN_GROWLITHE ferry\n"
```

Comment:
I assume the underscore here is a result of find/replace and probably not intended lol

Reply:
LMFAO Yeah I didn't care to unbreak everything lol